lookout/core/garbage_exclusion.py

```diff
@@ -0,0 +1,5 @@
+GARBAGE_PATTERNS = [
+    r"^.*[.-]min\.js"
```

Comment:
So this should go to `langs.javascript` package and loaded from all the languages

Reply:
As I mentioned in your last comment lookout doesn't differentiate by language so I think it's better to have a single list to be mindful of the interactions of the patterns for all languages at the same time.

Reply:
I don't want to update the global file for every language, this breaks the abstraction.

We do not care that lookout does not differentiate by language here at all.

1. get the list of packages under `lookout.style.format.langs`
2. take `garbage` module from each, take `GARBAGE_PATTERNS` from that module
3. `GARBAGE_PATTERNS = list(chain.from_iterable(...))`

Reply:
I'm obviously fine to do that but that means we'll have to handle those garbage files for every analyzer while we said that we should remove them globally. If we changed our mind I'll implement this analyzer contained version.

Reply:
The global `GARBAGE_PATTERNS` is still here in the same place in the global scope. This scanning code goes upper, that's all.

Reply:
Ok. I misunderstood because of the 1. that mentioned `lookout.style.format`. I'll put that in `lookout.core.langs` then.